web/src/people.ts

```diff
@@ -421,7 +421,7 @@ export function get_user_type(user_id: number): string | undefined {
 }
 
 export function emails_strings_to_user_ids_string(emails_string: string): string | undefined {
-    const emails = emails_string.split(",");
+    const emails = emails_string.split(",").map((email) => email.trim());
```

Comment:
Why this change as part of this commit?

Reply:
Yeah, I can just make a separate commit for it but is needed since `emails_strings_to_user_ids_string` is being called from other places which have extra space before `,` separator.